services/dataDeletionService.ts

```diff
@@ -1,11 +1,12 @@
 import * as Notifications from 'expo-notifications';
-import { db } from '../db';
+import { getDB } from '../db';
 import { periodDates, healthLogs, settings } from '../db/schema';
 import { eq, ne } from 'drizzle-orm';
 
 export class DataDeletionService {
```

Comment:
Tangential to this PR, but you should just remove the DB file. It's not a good idea to hand-pick which tables or even rows get deleted. You will most certainly be leaving data behind.

I suspect this won't be a trivial fix, so I suggest you create an issue to fix it later.